jboss/container/wildfly/s2i/legacy/artifacts/opt/jboss/container/wildfly/s2i/galleon/provisioning/generic_layers/pom.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    JBoss, Home of Professional Open Source
+    Copyright 2015, Red Hat, Inc. and/or its affiliates, and individual
+    contributors by the @authors tag. See the copyright.txt in the
+    distribution for a full listing of individual contributors.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jboss.galleon.s2i</groupId>
+        <artifactId>galleon-s2i-parent</artifactId>
+        <version>1.0.0.Final</version>
+    </parent>
+    <artifactId>layers-provisioning</artifactId>
+    <packaging>pom</packaging>
+    <name>Provision a set of Galleon layers from s2i feature-pack</name>
+  
+    <description>Provision a set of Galleon layers from s2i feature-pack</description>
+
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.html</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.jboss.galleon</groupId>
+                <artifactId>galleon-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>wildfly-provisioning-layers ${env.GALLEON_PROVISION_LAYERS}</id>
+                        <goals>
+                            <goal>provision</goal>
+                        </goals>
+                        <phase>compile</phase>
+                        <configuration>
+                            <install-dir>${project.build.directory}/server</install-dir>
```

Comment:
I'm a bit lost.

The installation dir for the server is hard-coded to `target/server` from this "temp" maven project which is then copied to `$JBOSS_HOME` later on by the script `galleon_replace_server`

but if we are using the `wildfly-maven-plugin` the server is installed in the `target/server` of the user's application pom.xml. Are we moving the server provisioned by the plugin to `$JBOSS_HOME` in another cekit module?

Reply:
The usage of wildfly-maven-plugin is not in this context, yes handled by the s2i module: https://github.com/wildfly/wildfly-cekit-modules/blob/v2/jboss/container/wildfly/s2i/bash/artifacts/opt/jboss/container/wildfly/s2i/s2i-core-hooks